packages/sprotty/src/features/context-menu/mouse-listener.ts

```diff
@@ -32,10 +32,9 @@ export class ContextMenuMouseListener extends MouseListener {
         super();
     }
 
-    mouseDown(target: SModelElement, event: MouseEvent): (Action | Promise<Action>)[] {
-        if (event.button === 2) {
-            this.showContextMenu(target, event);
-        }
+    contextMenu(target: SModelElement, event: MouseEvent): (Action | Promise<Action>)[] {
+        super.contextMenu(target, event);
```

Comment:
This seems unnecessary as the superclass does not do anything.

Reply:
Oh right, thanks! An old habit :)